monolithe/generators/lang/javascript/writers/apiversionwriter.py

```diff
@@ -12,6 +16,8 @@ def __init__(self, monolithe_config, api_info):
         output = monolithe_config.get_option("output", "transformer")
 
         self.output_directory = "%s/javascript/%s" % (output, api_info["version"])
+        self.enum_directory =  "%s/enums" % self.output_directory
```

Comment:
what about the abstract folder? Do you need to make sure that gets create as well?